##### 

IMU:
  # Default:9.81; gravity in the place your robot deployed
  gravity: 9.81               

DVL:
  # imu is reference frame
  T_I_D: 
  - [  -0.999928,   0.0116206, -0.00288846,   0.433828]
  - [  0.0115923,    0.999886,  0.00962664,   0.086180]
  - [      0.003,  0.00959246,   -0.999949,  -0.302024]
  - [        0.0,         0.0,         0.0,        1.0]

  # Default:0.0; Time offset from Temporal calibrtion between DVL and IMU: (t_imu = t_dvl + t_offset)
  timeoffset_I_D: -10.645267         
  # Default:1.0; Scale factor for DVL measurement effected by sound speed (caused by salinity, temperature...)
  scale: 1.0


###################################################################################
##### Those just from Kalib, maybe convert to msckf_dvio used reference frame #####
###################################################################################
CAM0:
  T_cam_imu:
  - [0.00786236726823823, 0.9999532121420647, 0.0056353090162310805, -0.1232206727449017]
  - [0.9999681294260654, -0.00785441912633285, -0.0014311646745676972, -0.310185995908388]
  - [-0.0013868356345183388, 0.005646381757991208, -0.9999830973871349, -0.27812151547998104]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [1]
  camera_model: pinhole
  distortion_coeffs: [-0.3352817921990125, 0.14104927301597872, -0.0007210633171553377,
    -0.00032715268767953613]
  distortion_model: radtan
  intrinsics: [1369.116784761009, 1368.4408514501083, 808.7406303246698, 620.3305937534344] # f_x, f_y, c_x, c_y 
  resolution: [1616, 1240]
  rostopic: /rov/sensors/stereo/left/image_decompressed
  timeshift_cam_imu: -0.09398272330912658
  timeoffset_I_C: 0.0
CAM1:
  T_cam_imu:
  - [0.011362662503790503, 0.9998882224979653, 0.00971763400653785, -0.043854791823369293]
  - [0.9999302479992342, -0.011330740940760782, -0.0033336835368381165, -0.3111845498969625]
  - [-0.003223202912534585, 0.009754835703046718, -0.9999472256791313, -0.2777771992579014]
  - [0.0, 0.0, 0.0, 1.0]
  T_cn_cnm1:
  - [0.9999855391661413, 0.0034948516557218013, -0.004087477278852761, 0.07931133771395357]
  - [-0.0034871782841654664, 0.9999921470868088, 0.00188291059996534, -0.0009070043568538431]
  - [0.004094025673476387, -0.0018686296095041489, 0.9999898735373106, 0.0002663456962428519]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0]
  camera_model: pinhole
  distortion_coeffs: [-0.33481309953528965, 0.14286781360099887, -0.0005321064881089663,
    -8.96731600981599e-05]
  distortion_model: radtan
  intrinsics: [1369.7017187790505, 1369.6206801121125, 810.8800453511235, 614.6826467473919]
  resolution: [1616, 1240]
  rostopic: /rov/sensors/stereo/right/image_decompressed
  timeshift_cam_imu: -0.2949693740418413